src/main/java/com/algorand/algosdk/account/Account.java

```diff
@@ -273,6 +274,29 @@ public Signature signBytes(byte[] bytes) throws NoSuchAlgorithmException {
         }
     }
 
+    /**
+     * Sign the given bytes, and wrap in Signature.
```

Comment:
Worth documenting that we prepend the message with a prefix for domain separation / security